src/lib/utility.ts

```diff
@@ -390,17 +390,12 @@ export const removeArticles = (str: string) => {
   return str
 }
 
-export const checkIfIdMatchesClipIdOrEpisodeId = (
+export const checkIfIdMatchesEpisodeIdOrAddByUrl = (
   id?: string,
-  clipId?: string,
   episodeId?: string,
   addByRSSPodcastFeedUrl?: string
 ) => {
-  return (
-    (clipId && id === clipId) ||
-    (!clipId && addByRSSPodcastFeedUrl && id === addByRSSPodcastFeedUrl) ||
-    (!clipId && episodeId && id === episodeId)
-  )
+  return (addByRSSPodcastFeedUrl && id === addByRSSPodcastFeedUrl) || (episodeId && id === episodeId)
```

Comment:
This looked very complicated to read initially. Its better now but would you mind changing it to:
```
let matches = false

if(addByRSSPodcastFeedUrl) {
    matches = addByRSSPodcastFeedUrl === id
} else if(episodeId){
    matches = episodeId === id
}

return matches
```

Reply:
Sure that's way better